medusa/server/web/config/general.py

```diff
@@ -99,6 +99,7 @@ def saveGeneral(self, log_dir=None, log_nr=5, log_size=1, web_port=None, notify_
         app.GIT_RESET = config.checkbox_to_value(git_reset)
         app.GIT_RESET_BRANCHES = helpers.ensure_list(git_reset_branches)
         app.GIT_PATH = git_path
+        config.change_GIT_PATH()
```

Comment:
@thraxis I would add a IF here if git path changed to avoid checking updates every config save. What do you think?